.evergreen/integration-tests.sh

```diff
@@ -61,6 +64,10 @@ if [ -z "$ORCHESTRATION_FILE" ]; then
    if [ "$SSL" != "nossl" ]; then
       ORCHESTRATION_FILE="${ORCHESTRATION_FILE}-ssl"
    fi
+
+   if [ "$LOAD_BALANCER" = "on" ]; then
```

Comment:
Confirming, given these conditionals are all setting the same `ORCHESTRATION_FILE` variable, it is expected that only _one_ of these sequence of conditionals are true in a given invocation of `integration-tests.sh`, correct?

Reply:
It is expected that more than one condition may be true. E.g. both `AUTH` and `SSL` conditions may be true to make `ORCHESTRATION_FILE`: `auth-ssl`.

FWIW this script is a modified copy of [run-orchestration.sh](https://github.com/mongodb-labs/drivers-evergreen-tools) from drivers-evergreen-tools. I prefer not diverging behavior too much to make a transition to drivers-evergreen-tools easier.

Reply:
Ah, I overlooked that these were appending to the `ORCHESTRATION_FILE` variable rather than overwriting it. Thank you and disregard.